apps/tx-signer/env/.env.mainnet

```
@@ -0,0 +1,2 @@
+REST_API_NODE_URL=https://rpc.akt.dev/rest
+RPC_NODE_ENDPOINT=https://rpc.akt.dev/rpc
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)